.github/workflows/ci.yml

```diff
@@ -113,12 +114,18 @@ jobs:
             test: test_autoscaler.sh
           - version: v1_15
             test: test_dynamic_config.sh
+          - version: v1_15
+            test: test_flink_operator_ha.sh
           - version: v1_16
             test: test_autoscaler.sh
           - version: v1_16
             test: test_dynamic_config.sh
+          - version: v1_16
+            test: test_flink_operator_ha.sh
           - version: v1_17
             test: test_dynamic_config.sh
+          - version: v1_17
+            test: test_flink_operator_ha.sh
```

Comment:
We should only test for one Flink version as the operator HA is not Flink version dependent

Reply:
Yes, that is why I exclude v1_15 to v1_17

Reply:
ah sorry, I thought this was includes. You are right